packages/core-components/src/components/AddElement/AddNewElement.vue

```diff
@@ -33,13 +33,14 @@
 
 <script lang="ts" setup>
 import type { ContentElement } from '@tailor-cms/interfaces/content-element';
+import some from 'lodash/some';
 
-const props = defineProps<{ library: any[]; allowedTypes: string[] }>();
+const props = defineProps<{ library: any; allowedTypes: any[] }>();
```

Comment:
Should we rename this one as it is no longer just Types but rather configurations (e.g. allowedElementConfig)